solution/backend/cmcs_regulations/settings/base.py

```diff
@@ -286,10 +269,3 @@
     "http://*.analytics.google.com",
 ]
 CSP_INCLUDE_NONCE_IN = ["script-src"]
-
-
-if DEBUG:
```

Comment:
this is for django toolbar to appear.  Moved to the local.py  file since we dont want this in prod nor do we want it on our test